packages/client/src/tools/client-utils.ts

```diff
@@ -313,6 +313,9 @@ export const getBlockExplorerUrlForTransaction = (
     case "POLKADOT":
       console.warn("[TODO]: Implement Block Explorer url for Polkadot");
       return "";
+    case "SKALE":
+      console.warn("[TODO]: Implement Block Explorer url for Skale"); // TODO: proably etherscan
```

Comment:
It will be etherscan, e.g. `https://etherscan.io/tx/0x7e4c5e1ae70562d9c5791f98d78d034427eae529a14a86ad608bd00ae8744012` where `0x7e4c5e1ae70562d9c5791f98d78d034427eae529a14a86ad608bd00ae8744012` is the transaction hash, which you will hopefully receive once the transaction has been submitted to the network.